website/screens/overview/introduction/IntroductionPage.tsx

```diff
@@ -185,6 +185,47 @@ const sections = [
       </>
     ),
   },
+  {
+    title: "Documentation of other releases",
+    content: (
```

Comment:
As it is not so oriented to releases, maybe it would be better a more general title like `Previous documentation` or something like this.

Reply:
We have links to next and current releases... Maybe we can call it `Other documentations`?

Reply:
Done!